datahub-web-react/src/graphql/contract.graphql

```diff
@@ -0,0 +1,54 @@
+fragment contractAssertionParameters on AssertionStdParameters {
+    value {
+        value
+        type
+    }
+    minValue {
+        value
+        type
+    }
+    maxValue {
+        value
+        type
+    }
+}
+
+fragment dataContractDetails on DataContract {
+    urn
+    type
+    properties {
+        freshness {
+            assertion {
+                ...assertionDetailsWithMonitors
```

Comment:
we should do `assertionDetails` not `assertionDetailsWithMonitors` as `monitors` are only for acryl version of datahub